docs/source/common/optimizers.rst

```diff
@@ -396,6 +396,7 @@ For example, here step optimizer A every batch and optimizer B every 2 batches.
         # update discriminator every 2 steps
         if optimizer_idx == 1:
             if (batch_idx + 1) % 2 == 0:
+                # closure/training_step will ONLY be called in this case
```

Comment:
```suggestion
                # The closure (which includes the `training_step`) won't run if this isn't called
```

Reply:
no, here it is getting called. 
the point of this is to say there is a missing "else" branch in this if clause where we intentionally omit the call to the step.

Reply:
```suggestion
                # The closure (which includes the `training_step`) won't run if the line below isn't executed
```

clearer?

Reply:
I would have preferred the else approach initially, but let's not get distracted by these details